docs/src/modules/components/ApiPage.js

```diff
@@ -270,7 +274,7 @@ import { ${componentName} } from '@mui/x-data-grid-pro';`;
       disableAd={false}
       disableToc={false}
       location={apiSourceLocation}
-      title={`${componentName} API – MUI`}
+      title={`${componentName} API`}
```

Comment:
This reproduces the content of `docs/src/modules/components/ApiPage` in MUI Core.